src/Apps/SaleAgreements/types.ts

```diff
@@ -0,0 +1,20 @@
+export type SaleAgreement = {
+  content: string | null | undefined
+  displayEndAt: string | null | undefined
+  displayStartAt: string | null | undefined
+  internalID: string
+  published: boolean
+  sale:
+    | {
+        internalID: string
+        isArtsyLicensed: boolean
+        isAuction: boolean | null | undefined
+        isBenefit: boolean | null | undefined
+        name: string | null | undefined
+      }
+    | null
+    | undefined
+  status: "CURRENT" | "PAST" | "ARCHIVED" | "%future added value"
+}
+
+export type SaleAgreements = SaleAgreement[]
```

Comment:
Most of these are non-null enforced at the [api and db layers](https://github.com/artsy/gravity/blob/main/db/schema.rb#L702-L712). But because of the [mp resolver helpers](https://github.com/artsy/metaphysics/blob/5ab40ff16bb1a4f40a25bec2f673904948c2a075/src/schema/v2/SaleAgreements/SaleAgreement.ts#L38-L41), these end up typed as nullable in our generated schema. 

Let me know if my understanding that there isn't a current way around this, other than not using the helpers.